.changelog/unreleased/improvements/2365-deduplicate-light-signature-checks.md

```diff
@@ -0,0 +1 @@
+- `[light]` Remove duplicated signature checks in light.VerifyNonAdjacent ([\#2365](https://github.com/cometbft/cometbft/issues/2365))
```

Comment:
no need for the duplicate changelog

Reply:
removed the one in breaking-changes since I have made this non-breaking